ChromProcess/Loading/peak/peak_from_csv.py

```diff
@@ -31,16 +31,38 @@ def peak_indices_from_file(peak_file, chromatogram):
     # chromatogram time axis.
     peaks_indices = np.empty(0, dtype="int64")
     for p_rt in peak_retention_times:
-        # Find the closest retention time value in to p_rt
-        delta_rt = np.abs(chromatogram.time - p_rt)
-        idx = delta_rt.argmin()
+        delta_rt = np.abs(chromatogram.time - float(p_rt))
```

Comment:
What is the reason for forcing a float conversion here? If the value is not a float already, there's a problem further upstream from this part of the code.

Reply:
The peak retention time is read in as a string, so we need to convert. Does the problem arise from the float conversion?